src/components/ResultsPage/ResultsPage.js

```diff
@@ -12,38 +12,40 @@ export default function ResultsPage({ watchByName, setWatchByName, watchesByType
 	}
    
 	return (
-		<div className="ResultsPage">
+		<div className="results-page">
 			<h1 className='header'>Watches That Meet Your Criteria:</h1>
-      {watchByName && (
-         <div>
-           <h2>Watch by Name:</h2>
-           <p>Name: {watchByName.name}</p>
-           <p>Type: {watchByName.type}</p>
-           <p>Maker: {watchByName.maker}</p>
-           <p>Cost: ${watchByName.cost}</p>
-           <p>Complications: {watchByName.complications}</p>
-           <p>Features: {watchByName.features}</p>
-           <p>Size: {watchByName.size}</p>
-         </div>
-       )}
-       {watchesByType.length > 0 && (
-         <div>
-           <h2>Watches by Type:</h2>
-           {watchesByType.map(watch => (
-             <p key={watch.id}>{watch.name}</p>
-           ))}
-         </div>
-       )}
-       {watchesByPrice.length > 0 && (
-         <div>
-           <h2>Watches by Price:</h2>
-           {watchesByPrice.map(watch => (
-             <p key={watch.id}>{watch.name}</p>
-           ))}
-         </div>
-       )}
-      <Link to={`/`} className='homePageLink'>
-        <button className='homePageButton' onClick={clearResults}>
+      <div className='all-results-container'>
+        {watchByName && (
+          <div>
+            <h2>Watch by Name:</h2>
+            <p>Name: {watchByName.name}</p>
```

Comment:
I like that you have a check here, to make sure that watchbyName has a value before you create the section